app/services/comment_service.rb

```diff
@@ -37,14 +37,13 @@ def self.destroy(comment:, actor:)
   end
 
   def self.update(comment:, params:, actor:)
-    new_mentions = comment.new_mentions_in(params[:body])
     comment.edited_at = Time.zone.now
     comment.body = params[:body]
 
     return false unless comment.valid?
     actor.ability.authorize! :update, comment
     comment.save!
 
-    EventBus.broadcast('comment_update', comment, new_mentions)
```

Comment:
Getting rid of the `new_mentions_in` method also means we can normalize the EventBus broadcasts to 'model, params, actor'

Eventually of course we want to move to named parameters, but for now this is a good thing.